src/server-extension/query/nft.ts

```diff
@@ -1,4 +1,21 @@
 export const passionQuery = `SELECT DISTINCT ne.issuer as id
 FROM nft_entity ne
 WHERE ne.current_owner = $1
-AND ne.current_owner != ne.issuer`
\ No newline at end of file
+AND ne.current_owner != ne.issuer`
+
+export const salesQuery = `SELECT
+    ne.id,
+    ne.name,
+    ne.issuer,
+    e.caller as buyer,
+    collection_id,
+    (e.meta::bigint) as sale_price,
+    e.timestamp,
+    me.image
+FROM
+    nft_entity ne
+    LEFT join event as e on e.nft_id = ne.id
+    LEFT join metadata_entity me on me.id = ne.metadata
+where
+    e.interaction = 'BUY'
+    and e.timestamp >= NOW() - INTERVAL '10 DAY'`
```

Comment:
missing new line here
any particular thing why have you picked 10 Days

Reply:
the new line is formatted by vscode.

I was testing on my db, havent run process for days, so `24 HOUR` would be no data.
will make it a parameter later.

Reply:
Have you used the dump?

Reply:
No, didnt know that and havent yet. usually run the process to update db.

Reply:
Oh, I see, found out at README, thanks.